py_cui/control_widgets/slider.py

```diff
@@ -153,20 +153,25 @@ def _custom_draw_border(self, start_y: int):
         renderer = self.get_renderer()
         ui_element = self
```

Comment:
Out of curiosity, is there a reason for assigning `ui_element` to `self` instead of just using `self` throughout?

Reply:
I expected that ```TextBlock``` will need this height control someday, and since drawing logic is very similar for both - I assumed that someday you're going to separate this method and move to new mixin class that's inherited by both. For that reason I tried to leave the variable name as-is.

Let me know if you want to change those!